File: recrutement/src/main/java/com/ant/recrutement/entities/Experience.java
package com.ant.recrutement.entities;
import lombok.Data;

import javax.persistence.*;
import java.util.Date;

@Entity
@Data
public class Experience {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String titre;
    private String societe;
    private String description;
    @Temporal(TemporalType.DATE)
    private Date debut;
    @Temporal(TemporalType.DATE)
    private Date fin;


    @ManyToOne
    private Cv cv;

    @ManyToOne
    private TypePoste typePoste;
}

File: recrutement-front/src/app/shared/model/experience.ts
import {Cv} from './cv';
import {TypePoste} from './type-poste';

export class Experience {
  id: number;
  titre: string;
  societe: string;
  debut: any;
  fin: any;
  cv: Cv;
  typePoste: TypePoste;
  description: string;
}
